x-pack/plugins/security_solution/public/common/components/ml_popover/jobs_table/jobs_table.tsx

```diff
@@ -82,16 +81,24 @@ const getJobsTableColumns = (
   },
   {
     name: i18n.COLUMN_GROUPS,
-    render: ({ groups }: SecurityJob) => (
-      <EuiFlexGroup wrap responsive={true} gutterSize="xs">
-        {groups.map((group) => (
-          <EuiFlexItem grow={false} key={group}>
-            <EuiBadge color={'hollow'}>{group}</EuiBadge>
-          </EuiFlexItem>
-        ))}
-      </EuiFlexGroup>
-    ),
-    width: '140px',
+    render: ({ groups }: SecurityJob) => {
+      const renderItem = (group: string, i: number) => (
+        <EuiBadge color="hollow" key={`${group}-${i}`} data-test-subj="group">
+          {group}
+        </EuiBadge>
+      );
+
+      return (
+        <PopoverItems
+          items={groups}
+          numberOfItemsToDisplay={0}
+          popoverButtonTitle={`${groups.length} Groups`}
+          renderItem={renderItem}
+          dataTestPrefix="groups"
+        />
```

Comment:
Don't mind this... 😅 


I was testing https://github.com/elastic/kibana/pull/131166 the other day and had it with the badge overflow. Since in doing this PR I learned of the `PopoverItems` component I figured there was a nice quick fix here:


<p align="center">
  <img width="300" src="https://user-images.githubusercontent.com/2946766/169757645-3480bacd-952c-4892-a32b-c3c072de2407.png" /><img width="300" src="https://user-images.githubusercontent.com/2946766/169757477-2c4ca8b0-2957-4131-b301-dbb298f434fa.png" />
</p>

cc @randomuserid